### PR TITLE
Ensure that VB XML doc comment completion returns correct tag names

### LINKED ISSUE #8546: VB XML doc comment completion incorrectly suggests 'return' tag (should be 'returns' with an 's')
(Filing this on behalf of Jan (Đonny) Záruba)
csxmlcompletion
vbxmlcompletion

"So, is it <returns> or <return>. OK, I know it's <returns>, but in that case there is a bug in VB IntelliSense.
"I was not playing with VBXMLDoc.xml file (if it is still supported)."

### PR DESCRIPTION
**Pull request by DustinCampbell**
Fixes #8546 

Note: The bug was trivial, but I went ahead and created constants for all of the tag and attribute name string literals sprinkled throughout XML doc comment completion to avoid bugs like these in the future.

tagging @dotnet/roslyn-ide

### CODE REVIEW

File: src/EditorFeatures/VisualBasicTest/Completion/CompletionProviders/XmlDocCommentCompletionProviderTests.vb [hunk 1]
```diff
@@ -33,190 +31,190 @@ Namespace Tests
...
-    ''' &lt;$$
-    ''' &lt;/summary&gt;
+    ''' <summary>
+    ''' <$$
+    ''' </summary>
     Sub Foo()
     End Sub
 End Class
-</File>.Value
+"
```
Comment (Pilchie) [inline src/EditorFeatures/VisualBasicTest/Completion/CompletionProviders/XmlDocCommentCompletionProviderTests.vb]: Much more readable - thanks!

**Comment by Pilchie** [comment]
:+1:

**Comment by rchande** [comment]
:+1:

**Comment by jasonmalinowski** [comment]
:+1:

**Pull request merged**